lib/std/os/uefi.zig

```diff
@@ -150,3 +150,8 @@ test "GUID formatting" {
 
     try std.testing.expect(std.mem.eql(u8, str, "32cb3c89-8080-427c-ba13-5049873bc287"));
 }
+
+comptime {
```

Comment:
I've gone ahead and implemented both of these, it does however mean that uefi will continue to be untested on the CI. I can however confirm that all of the tests in uefi pass at the moment so we can get merge this PR and have a working std.os.uefi in 0.10 (at the least with debug info stripped).

Reply:
We can still add CI coverage for UEFI - let's just do it intentionally, via a new entry in the test matrix:

https://github.com/ziglang/zig/blob/20925b2f5c5c0ae20fdc0574e5d4e5740d17b4d6/test/tests.zig#L40

As you noted, I think it would be best to do that in a separate contribution so that we can get this particular PR landed in time for 0.10.0.

Reply:
I did attempt to add uefi to the test targets, however a lot of the tests (behavior was the first that failed) that rely on the std.os interface (which uefi pretty much doesn't implement) were causing failures. Is there any way to get uefi to *only* run stdlib tests?

Reply:
On a side note: I have *absolutely* no idea what is causing the compile error for debug on ci.ziglang.org